config/esw.yaml

```diff
@@ -55,60 +74,109 @@ brushed_motors:
       kP: 0.05
       kI: 0.0
       inversion: -1.0
+      enable_limit_a: false
+      enable_limit_b: false
+      limit_a_is_fwd: true
+      calibration_vel: 0.25
     - name: "finger"
       driver_voltage: 12.0
       motor_max_voltage: 12.0
       mcu_id: 1
       motor_id: 3
       inversion: 1.0
+      enable_limit_a: true
+      enable_limit_b: true
+      counts_limit_a: 0
+      counts_limit_b: 0
+      limit_a_is_fwd: true
+      calibration_vel: 0.25
     - name: "gripper"
       driver_voltage: 12.0
       motor_max_voltage: 12.0
       mcu_id: 1
       motor_id: 2
       inversion: 1.0
+      enable_limit_a: false
+      enable_limit_b: false
+      limit_a_is_fwd: true
+      calibration_vel: 0.25
     - name: "mast_gimbal_up_down"
       driver_voltage: 12.0
       motor_max_voltage: 12.0
       mcu_id: 2
       motor_id: 4
       inversion: -1.0
+      enable_limit_a: false
+      enable_limit_b: false
+      limit_a_is_fwd: true
+      calibration_vel: 0
```

Comment:
Thought I commented this on my first pass but why do we need all 4 variables here? When both the first two are false, can't we just skip all the rest of the variables?

Reply:
Good point